flow360/component/simulation/primitives.py

```diff
@@ -7,6 +7,18 @@
 from flow360.component.simulation.framework.entity_base import EntityBase
 
 
+class ReferenceGeometry(Flow360BaseModel):
+    """
+    Contains all geometrical related refrence values
+    Note:
+    - mesh_unit is removed from here and will be a property
+    """
+
+    moment_center: Optional[LengthType.Point] = pd.Field()
+    moment_length: Optional[Union[LengthType.Positive, LengthType.Moment]] = pd.Field()
```

Comment:
do we store float, or (float, float, float) depending what user provided, or we auto convert? The latter would be easier for solver, I think?

Reply:
yes, we convert to (float float float) in maybe a after validator.